PyRDF/backend/Spark.py

```diff
@@ -0,0 +1,67 @@
+from __future__ import print_function
+from .Dist import Dist
+from pyspark import SparkConf, SparkContext
+
+class Spark(Dist):
+    """
+    Backend that executes the computational graph
+    using using `Spark` framework for distributed
+    execution.
+
+    """
+    def __init__(self, config = {}):
+        """
+        Creates an instance of the Spark
+        backend class.
+
+        Parameters
+        ----------
+        config : dict (optional)
+            The config options for Spark backend. The default
+            value is an empty Python dictionary `{}`. Config
+            specifically reads three options, 'SparkContext'
+            or 'SparkConf' and 'npartitions'. If 'SparkContext'
+            is not given, 'SparkContext' uses 'SparkConf' option,
+            which by default is 'None' and 'npartitions' is
+            '2' by default.
+
+        """
+        super(Spark, self).__init__(config)
+
+        try:
+            # Remove the value of 'npartitions' from config dict
+            self.npartitions = config.pop('npartitions')
+        except KeyError:
+            # Set the default value to 2 if 'npartitions'
+            # doesn't exist
+            self.npartitions = config.get('spark.executor.instances') or 2
```

Comment:
Again, if a previous `SparkContext` is loaded, this value might not be the correct one...
Can't you just move this after line 40 and *then* you try to get the value from the `SparkConf` set in the `SparkContext`?

Reply:
Done @diocas 👍